src/jollyjumper/rules.py

```diff
@@ -14,10 +24,158 @@ def get_sirrematic_enjambment(previous_token, next_token):
                          ]
     while sirremactic_pairs:
         sirrematic_pair = sirremactic_pairs.pop()
-        if sorted((previous_token.pos_, next_token.pos_)) == sorted(sirrematic_pair)\
-                and (next_token.is_ancestor(previous_token) or previous_token.is_ancestor(next_token)):
+        if sorted((previous_token.pos_, next_token.pos_)) == sorted(
+                sirrematic_pair) \
+                and (next_token.is_ancestor(previous_token)
+                     or previous_token.is_ancestor(next_token)):
             return sirrematic_pair
     return None
 
 
-rules = locals()
\ No newline at end of file
+def get_sirrematic_relation_words_preposition_enjambment(previous_token,
+                                                         next_token):
+    """
+    Checks if sirrematic enjambment exists between two lines
+    :param previous_token: The word before a newline character
+    :return: Sirrematic type or None if not found
+    """
+    if previous_token.lower_ in ENJAMBMENT_PREPOSITIONS and re.search(
+            'AdpType=Prep', previous_token.tag_):
+        return [previous_token.pos_, next_token.pos_]
+    return None
```

Comment:
Try using `return previous_token.pos_, next_token.pos_` and replace where needed.